amr-wind/turbulence/LES/OneEqKsgs.cpp

```diff
@@ -12,6 +12,7 @@ namespace amr_wind {
 namespace turbulence {
 
 template <typename Transport>
+// cppcheck-suppress uninitMemberVar
```

Comment:
@jrood-nrel  do you remember what member it was complaining about here?

Reply:
Sure. Here is the list I was working on: https://github.com/Exawind/amr-wind/runs/4717856939?check_suite_focus=true

Reply:
Interesting, it complains about members that seem to be initialized:

<img width="886" alt="Screen Shot 2022-01-06 at 1 42 19 PM" src="https://user-images.githubusercontent.com/433480/148449015-b3bee698-22da-478d-9143-f18b0a555b1c.png">

https://github.com/Exawind/amr-wind/blob/dc887e210c9ad99e2fd7225a6d91b9826c350d72/amr-wind/turbulence/LES/OneEqKsgs.cpp#L14-L26

Reply:
Yeah it seems to get false positives when the initialization is complex.